src/sdkV2.ts

```diff
@@ -233,10 +248,28 @@ export default class CegaEvmSDKV2 {
     );
   }
 
-  // TODO: add this method when new contracts are deployed
-  // async dcsWithdrawStuckAssets() {
-  //   return null;
-  // }
+  /**
+   * Withdraws "stuck" funds from the Treasury
+   * This method must be called after withdraw queues are processed
+   * for native ETH underlying vaults, for non EOA wallet (eg. multisigs)
+   *
+   * @param asset - The address of asset to be withdrawn
+   * @param receiver - The address of the wallet to receive funds
```

Comment:
`receiver` isn't a param in this case
this function only has `asset` as a param

unless you want to add `receiver = null` as an optional param, and then use that in the SC function call if not null (i.e. allow the sdk to call a receiver other than itself)